docs/jam/jam/introduction.rst

```diff
@@ -10,6 +10,8 @@ Jam is a language frontend that translates to
 :ref:`Lekvar <lekvar-introduction>`, which can then be compiled or interpreted
 by the backend.
 
+Each section will feature a worded description, an EBNF definition and where possible, coded examples to demonstrate that feature in Jam syntax.
```

Comment:
just `a more formal syntax definition` would be fine. Metasyntax is the syntax of a metalanguage, a language used to describe a language.